src/node_perf.cc

```diff
@@ -489,29 +489,57 @@ ELDHistogram::ELDHistogram(
                           Histogram(1, 3.6e12),
                           resolution_(resolution) {
   MakeWeak();
-  uv_timer_init(env->event_loop(), &timer_);
+  CHECK_EQ(uv_timer_init(env->event_loop(), &timer_), 0);
```

Comment:
should this be in an `else` below?

Reply:
Excellent question. I'll add a comment to the code to clarify why it should be as it is here.